qiskit/pulse/schedule.py

```diff
@@ -372,17 +372,22 @@ def draw(self, dt: float = 1, style: Optional['SchedStyle'] = None,
             table: Draw event table for supported commands
             label: Label individual instructions
             framechange: Add framechange indicators
+            scaling: Deprecated, see `scale_factor`
 
         Returns:
             matplotlib.figure: A matplotlib figure object of the pulse schedule.
         """
         # pylint: disable=invalid-name, cyclic-import
+        if scaling:
+            warnings.warn('The parameter "scaling" is being replaced by "scale_factor"'
+                          '', DeprecationWarning, 3)
+            scale_factor = scaling
 
         from qiskit import visualization
 
         return visualization.pulse_drawer(self, dt=dt, style=style,
                                           filename=filename, interp_method=interp_method,
-                                          scaling=scaling, channels_to_plot=channels_to_plot,
+                                          scaling=scale_factor, channels_to_plot=channels_to_plot,
```

Comment:
I believe we should also update this in the pulse visualization module. The same for Schedule.

Reply:
Alright then, I wasn't too sure I should do that before now.
Thanks.